batch/batch/server/database.py

```diff
@@ -20,26 +14,30 @@ def create_synchronous(config_file):
 
 
 class JobsTable(Table):
+    uri_log_mapping = {'input': 'input_log_uri',
+                       'main': 'main_log_uri',
+                       'output': 'output_log_uri'}
+
     async def __init__(self, db, name='jobs'):
         schema = {'id': 'BIGINT NOT NULL AUTO_INCREMENT',
                   'state': 'VARCHAR(40) NOT NULL',
                   'exit_code': 'INT',
                   'batch_id': 'BIGINT',
-                  'scratch_folder': 'VARCHAR(1000)',
-                  'pod_name': 'VARCHAR(1000)',
+                  'scratch_folder': 'VARCHAR(1024)',
+                  'pod_name': 'VARCHAR(1024)',
                   'pvc': 'TEXT(65535)',
                   'callback': 'TEXT(65535)',
                   'task_idx': 'INT NOT NULL',
                   'always_run': 'BOOLEAN',
+                  'cancelled': 'BOOLEAN',
                   'time_created': 'TIMESTAMP DEFAULT CURRENT_TIMESTAMP',
-                  'time_ended': 'TIMESTAMP',
-                  'user': 'VARCHAR(1000)',
+                  'duration': 'BIGINT',
```

Comment:
I put huge defaults on these for text. Not sure this is necessary and may be allocating a ton of space.